packages/ancoms-roms/package.py

```diff
@@ -0,0 +1,149 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import sys
+
+from spack.package import *
+
+
+class AncomsRoms(MakefilePackage):
+    """ROMS is a free-surface, terrain-following,
+    primitive equations ocean model widely used by
+    the scientific community for a diverse range of applications"""
+
+    homepage = "https://www.myroms.org/"
+    git = "https://github.com/ACCESS-Community-Hub/roms"
+
+    version("coastri", branch="unsw-coastal-regional-lab", preferred=True)
```

Comment:
Sorry to be a pain, but I think it would be better for the version to be `unsw`, as we may more than one CoastRI group wanting to have their own version of ROMS.